robots.txt

```diff
@@ -1,3 +1,5 @@
 User-agent: *
 Disallow: /source/old
 Disallow: err404.html
+
+Sitemap: https://www.openssl.org/sitemap.xml
```

Comment:
There is missing EOL at EOF here.

Reply:
I have added a new line in the end and committed but it is not visible (somehow github not showing)